src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/returncount/Example1.java

```diff
@@ -0,0 +1,47 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="ReturnCount">
+      <property name="max" value="3"/>
+    </module>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.returncount;
+
+import java.util.function.Predicate;
+
+// xdoc section -- start
+public class Example1 {
+    public Example1() {}
+
+    // ok below, because it doesn't exceed 1 return for default void restriction
+    public Example1(int i) { return; }
```

Comment:
Please make only single line between methods.
If violation, place a comment ( no empty lines), if no violation - it will be single empty line.

Reply:
`// ok below, because default void restriction is 1`

I like this wording better. If you agree please apply same style to other ok comments 
https://checkstyle.org/checks/coding/returncount.html#Properties

Reply:
Done